### PR TITLE
RSG-Group renamed to just RSG

### PR DESCRIPTION
**Pull request by radi-cho**

### CODE REVIEW

**Comment by radi-cho** [comment]
What's happening with this, @retrixe ?

File: static/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "RSG-Group",
-  "name": "RSG-Group Webpage",
+  "short_name": "RSG",
```
Comment (retrixe) [inline static/manifest.json]: Undo this.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![js-standard-style](https://cdn.rawgit.com/standard/standard/master/badge.svg)](https://github.com/standard/standard)
 
-(Un)official RSG-Group website built in Material Design w/ Next.js w/ Express w/ Apollo (WIP) w/ SSR w/ Babel w/ Redux (MobX under consideration) w/ Flow (whew.)
+(Un)official RSG website built in Material Design w/ Next.js w/ Express w/ Apollo (WIP) w/ SSR w/ Babel w/ Redux (MobX under consideration) w/ Flow (whew.)
```
Comment (retrixe) [inline README.md]: Change this to Official, would be good.

Comment (retrixe) [inline README.md]: Can you change this to Official please?

Comment (retrixe) [inline README.md]: Test.

File: static/manifest.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  "name": "RSG's Webpage",
```
Comment (retrixe) [inline static/manifest.json]: Please undo changes to this file, it will cause performance impacts in certain cases.
Anyways, it doesn't look right.

Comment (retrixe) [inline static/manifest.json]: Test.

Comment (retrixe) [inline static/manifest.json]: Test.

File: README.md (hunk 2 above)
Comment (radi-cho) [reply README.md]: Okay

Comment (radi-cho) [reply README.md]: what test?

File: static/manifest.json (hunk 3 above)
Comment (radi-cho) [reply static/manifest.json]: I'll undo this

**Comment by radi-cho** [comment]
Is this PR ready to be merged?

**Pull request merged**